bundles/org.openhab.ui/web/src/components/item/group-form.vue

```diff
@@ -8,9 +8,9 @@
               {{ type }}
             </option>
           </optgroup>
-          <optgroup label="Numbers with Dimensions">
-            <option v-for="dimension in types.Dimensions" :key="dimension" :value="'Number:' + dimension" :selected="item.groupType === 'Number:' + dimension">
-              {{ 'Number:' + dimension }}
+          <optgroup label="Numbers with Dimensions (System Unit)">
```

Comment:
Not sure why we need this (I don't understand the addition and it would in some cases not fit, or barely):

![image](https://github.com/openhab/openhab-webui/assets/2004147/728a79bd-b55c-45b4-a2d7-c7b936d96f89)

Reply:
I have added `(System Unit)` to explain the unit which is displayed in the picker, but we can remove it.

Reply:
I understand, we probably should have a system that better reflects how the underlying things work though - i.e. remove dimensions from the type picker altogether, and have another picker for the dimension appear when Number is picked.

Reply:
Agreed. I have implemented and pushed this!

Reply:
That was quick :sweat_smile: